src/sig_stfl/xmss/sig_stfl_xmss_sha256_h10.c

```diff
@@ -56,6 +56,10 @@ OQS_SIG_STFL_SECRET_KEY *OQS_SECRET_KEY_XMSS_SHA256_H10_new(void) {
 	sk->sigs_left = NULL;
 	sk->sigs_total = NULL;
 
+	// Secret serialize/deserialize function
+	sk->serialize_key = OQS_SECRET_KEY_XMSS_serialize_key;
+	sk->deserialize_key = OQS_SECRET_KEY_XMSS_deserialize_key;
+
 	// Initialize the key with length_secret_key amount of bytes.
 	sk->secret_key_data = (uint8_t *)malloc(sk->length_secret_key * sizeof(uint8_t));
```

Comment:
Check for NULL after malloc.

Reply:
fixed in https://github.com/open-quantum-safe/liboqs/pull/1542/commits/67d3aebf4c0574faa7a754373cb21129a3a37073